selenium.go

```diff
@@ -223,6 +225,37 @@ const (
 	SameSiteEmpty           = ""
 )
 
+type PointerType string
+
+const (
+	MousePointer PointerType = "mouse"
+	PenPointer               = "pen"
+	TouchPointer             = "touch"
+)
+
+//controls how the offset for the pointer move action is calculated
```

Comment:
Begin the comment sentence with the name being described. Here:

// PointerMoveOrigin controls...

https://golang.org/doc/effective_go#commentary